finish/src/main/liberty/config/server.xml

```diff
@@ -11,22 +11,39 @@
     httpPort="${server.http.port}" 
     httpsPort="${server.https.port}" />
 
+  <!-- tag::location[] -->
   <include location="userRegistry.xml"/>
+  <!-- end::location[]-->
 
   <application location="${application.name}" 
                type="war"
                id="${application.name}"
                name="${application.name}" context-root="/">
+    <!-- tag::application-bnd[] -->
     <application-bnd>
+      <!-- tag::security[] -->
+      <!-- tag::security-role-admin[] -->
       <security-role name="admin">
+      <!-- end::security-role-admin[] -->
+        <!-- tag::group[] -->
+        <!-- tag::group-name-Manager[] -->
         <group name="Manager" />
+        <!-- end::group-name-Manager[] -->
+        <!-- tag::group-name-TeamLead[] -->
         <group name="TeamLead" />
+        <!-- end::group-name-TeamLead[] -->
+        <!-- end:: group[] -->
       </security-role>
+      <!-- tag::security-role-user[] -->
       <security-role name="user">
+      <!-- end::security-role-user[] -->
+        <!-- tag::group-name-Employee[] -->
         <group name="Employee" />
+        <!-- end::group-name-Employee[] -->
       </security-role>
+      <!-- end::security[] -->
     </application-bnd>
   </application>
-
+ <!-- end::application-bnd[] -->
```

Comment:
- hotspot is highlighting extra line:
    `<!-- end::application-bnd[] -->` should be before `</application>` and after `</application-bnd>`